src/Containers/KitchenStatusModel/KitchenStatusModel.stories.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { Meta, Story } from '@storybook/react';
+import { KitchenStatusModel, KitchenStatusModelProps } from '../../index';
+import { createStoryTitle} from '../../Constants';
+
+export default {
+    title: createStoryTitle('KitchenStatusModel'),
+    component: KitchenStatusModel,
+    argTypes: {
+        onPauseClick: {action: 'Paused For'},
+        onStatusClick: {action: 'Current Status'},
+        onUpdateClick: {action: 'Update Status'}
+    },
+    args: {
+        minuteAmounts: [10, 20, 30, 40],
+        statusBox: [
+            {
+                status: 0,
+                color: 'green',
+                header: 'Normal',
+                body: 'Business as usual',
+            },
+            {
+                status: 1,
+                color: 'yellow',
+                header: 'Busy',
+                body: 'Need more prep time',
+            },
+            {
+                status: 2,
+                color: 'red',
+                header: 'Pause',
+                body: 'Pause new orders',
+            },
```

Comment:
You need onClicks for the 3 Buttons as well as OnClicks for the Pause for X minutes buttons